readme.md

```diff
@@ -358,10 +358,8 @@ Sets `options.method` to the method name and makes a request.
 
 #### got.extend([options])
 
-Configure a new `got` instance with default `options` and (optionally) a custom `baseUrl`:
+Configure a new `got` instance with default `options`. `options` are merged with the extended instance's `defaults.options` as described in [`got.assignOptions`](#got.assignoptionsparentoptions,-newoptions).
```

Comment:
Stop guessing. You need to change the URL to `#gotassignoptionsparentoptions-newoptions`.

Reply:
Glad you've taken my tip ^_^